packages/cli/src/commands/versions/bump.ts

```diff
@@ -55,7 +56,16 @@ export default async () => {
   // Track package versions that we want to remove from yarn.lock in order to trigger a bump
   const unlocked = Array<{ name: string; range: string; target: string }>();
   await workerThreads(16, dependencyMap.entries(), async ([name, pkgs]) => {
-    const target = await findTargetVersion(name);
+    let target: string;
+    try {
+      target = await findTargetVersion(name);
+    } catch (error) {
+      if (error instanceof NotFoundError) {
```

Comment:
I prefer using `error.name === 'NotFoundError'` as a more resilient check

Reply:
@ahrberg seems you still have this one to address

Reply:
@freben sorry missed that one